graphql/types/Product.graphql

```diff
@@ -348,6 +353,10 @@ type Discount {
   Discount name
   """
   name: String
+  """
+  Additional Info
+  """
+  additionalInfo: [AdditionalInfo]
```

Comment:
Yes, the resolvers will consider the offers so in this case it will return correctly, and as it should.

Reply:
Got it, thanks!

Reply:
The return is optional? `[AdditionalInfo]` means that it could return null or null values inside the array.

If it always gonna be an array with non-nullable values, should be: `[AdditionalInfo!]!`
If it can be null, but the values inside the array don't, should be: `[AdditionalInfo!]`